packages/typeguards/package.json

```diff
@@ -0,0 +1,50 @@
+{
+	"$schema": "http://json.schemastore.org/package",
+	"name": "@discordjs/typeguards",
+	"version": "0.1.0-canary.0",
+	"description": "Typeguard helpers for Discord.js types.",
+	"contributors": [
+		"Suneet Tipirneni <suneettipirneni@icloud.com>",
+		"NotSugden <email>"
+	],
+	"license": "Apache-2.0",
+	"scripts": {
+		"test": "echo \"Error: run tests from root\" && exit 1",
+		"build": "tsc --build --force"
+	},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/discordjs/discord.js-modules.git"
+	},
+	"bugs": {
+		"url": "https://github.com/discordjs/discord.js-modules/issues"
+	},
+	"homepage": "https://github.com/discordjs/discord.js-modules/tree/main/packages/typeguards",
+	"keywords": [
+		"discord",
+		"typeguards",
+		"typescript",
+		"discordapp",
+		"discordjs"
+	],
+	"main": "dist/index.js",
+	"directories": {
+		"lib": "src",
+		"test": "__tests__"
+	},
+	"files": [
+		"dist"
+	],
+	"dependencies": {
+		"discord.js": "^13.1.0"
```

Comment:
This shouldn't be a dep. If you only use it in tests (which is the only place you should use it, it should be a devDep)